salt/modules/junos.py

```diff
@@ -1310,3 +1320,117 @@ def commit_check():
         ret['out'] = False
 
     return ret
+
+
+def get_table(table, file, path=None, target=None, key=None, key_items=None,
+              filters=None, args=None):
+    """
```

Comment:
Can you change the `"""` references to use `'''`? We like to use the single quotes instead of double quotes for doc strings.

This new function could also use a `.. versionadded:: Fluorine` tag too. :)